File: packages/db/db_reload.py
import asyncio
from prisma import Prisma

async def main():
    prisma = Prisma()
    await prisma.connect()
    await prisma.execute_raw("NOTIFY pgrst, 'reload schema';")
    print("Schema reloaded.")
    await prisma.disconnect()

if __name__ == '__main__':
    asyncio.run(main())

File: packages/db/db_reload.js
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function main() {
  await prisma.$executeRawUnsafe(`NOTIFY pgrst, 'reload schema'`);
  console.log('Reloaded PostgREST schema cache');
}

main()
  .catch(console.error)
  .finally(() => prisma.$disconnect());
